### PR TITLE
tests_l2: Update README 

### PR DESCRIPTION
**Pull request by vbedida79**
Update tests/l2 readme 

- word dgpu to gpu and Intel® Data Center GPU Flex Series
- word workload to testcase
- SGX to Intel® SGX

Signed-off-by: Veenadhari Bedida [veenadhari.bedida@intel.com](mailto:veenadhari.bedida@intel.com)

### CODE REVIEW

**Comment by vbedida79** [comment]
This will be resolved thru interval review and a new PR will be submitted.

**Comment by uMartinXu** [comment]
Please add PR description.

File: tests/l2/README.md [hunk 1]
```diff
@@ -16,7 +19,7 @@ oc get pods | grep clinfo
 oc logs <clinfo_pod_name>
 ```
 
-A sample result for clinfo detecting dGPU card:
+A sample result for clinfo detecting GPU card:
 ```
```
Comment (uMartinXu) [inline tests/l2/README.md:23]: Intel data center GPU

File: tests/l2/README.md [hunk 2]
```diff
@@ -47,7 +50,7 @@ Number of devices                                 1
...
 
 #### SGX test case
-The test case used is SGX SDK [Sample Enclave App](https://github.com/intel/linux-sgx/tree/master/SampleCode/SampleEnclave), which launches a simple SGX enclave. Similar to dGPU,  OCP buildconfig and imagestream are leveraged for the container image.
+The test case used is SGX SDK [Sample Enclave App](https://github.com/intel/linux-sgx/tree/master/SampleCode/SampleEnclave), which launches a simple SGX enclave. Similar to GPU,  OCP buildconfig and imagestream are leveraged for the container image.
 The job pod is scheduled on a node enabled with SGX requesting enclave memory resource ```sgx.intel.com/epc```. The resource is created by the SGX device plugin.
```
Comment (uMartinXu) [inline tests/l2/README.md:54]: Intel data center GPU

File: tests/l2/README.md [hunk 3]
```diff
@@ -1,13 +1,16 @@
 ### L2 overview
 This layer consists of workloads for resource provisioning after Intel Device Plugins Operator is installed and custom resources are created.
 
-#### dGPU testing workload
-The workload used is [clinfo](https://github.com/Oblomov/clinfo), which displays the related information of dGPU card. The OCP buildconfig is leveraged to build clinfo container image and push it to the embedded repository through OCP imagestream.
-The Job pod is scheduled on a node with dGPU card and the resource ```gpu.intel.com/i915``` is registered by the dGPU device plugin.
-Below operations are verified on OCP-4.11 bare-metal cluster.
-To deploy the workload:
+#### GPU testcase
+The testcase used is [clinfo](https://github.com/Oblomov/clinfo), which displays the related information of GPU card. The OCP buildconfig is leveraged to build clinfo container image and push it to the embedded repository through OCP imagestream.
```
[the remark this answers is not in the document]
Comment (vbedida79) [reply tests/l2/README.md:5]: Can we only add it to the first occurrence? And call it GPU in later occurrences

File: tests/l2/README.md (hunk 2 above)
Comment (vbedida79) [reply tests/l2/README.md:54]: changed it in previous occurrences

File: tests/l2/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vbedida79) [reply tests/l2/README.md]: updated

File: \ [hunk 4]
```diff
@@ -0,0 +1,23 @@
+
+tests_l3: Update README
```
[the remark this answers is not in the document]
Comment (vbedida79) [reply \]: My bad, updated it

[the remark this answers is not in the document]
Comment (vbedida79) [reply \]: Added (r) only for first occurrence in readme. Rest all occurrences don't have it.

**Comment by vbedida79** [comment]
Will be updated in a new PR for 1.0.0 release.

**Pull request closed without merging**